hail/python/hailtop/utils/filesize.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any
+
+
+_PREFIXES = ["B", "K", "M", "G", "T", "P", "E", "Z", "Y"]
+
+
+def filesize(size: Any) -> str:
```

Comment:
This function would blow up on an input that is not coercible to an `int`, so it would be nice to narrow the signature just to those valid inputs. AFAIK, that's unfortunately not expressible in the python type system, but from the usage here it looks like we ever only use this function on `int`s anyway, so I think we should just enforce that the input be `int` and remove the cast.

Reply:
good call, i think i just got caught up in trying to replicate what `hurry.filesize` does haha, fixed!